packages/office-ui-fabric-react/src/components/Nav/Nav.types.ts

```diff
@@ -11,6 +11,14 @@ export interface INav {
    * all groups of the Nav must have populated key properties.
    */
   selectedKey: string | undefined;
+
+  /**
+   * Sets focus to the first tabbable item in the zone.
+   * @param forceIntoFirstElement - If true, focus will be forced into the first element, even
+   * if focus is already in the focus zone.
+   * @returns True if focus could be set to an active element, false if no operation was taken.
+   */
+  focus?: (forceIntoFirstElement?: boolean) => boolean;
```

Comment:
Change this one as optional to pass build, otherwise we need change Nav.Base.tsx in other different project only to pass the build.

Reply:
I don't see anything like that in Fabric. Do you mean somewhere else?

Reply:
https://github.com/OfficeDev/office-ui-fabric-react/blob/5.0/packages/experiments/src/components/Nav/NavBase.tsx, where it used INav declared at packages\office-ui-fabric-react\src\components\Nav\Nav.types.ts

Reply:
@ecraig12345  Hi Elizabeth Could you help me to take a look, thanks.